src/In.ProjectEKA.OtpService/Otp/Logger/Log.cs

```diff
@@ -0,0 +1,44 @@
+
+namespace In.ProjectEKA.OtpService.Otp.Logger
+{
+    using System;
+    using Serilog;
+    
+    public class Log
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4954915)